services/GraphQL/queries/users.ts

```diff
@@ -1,73 +1,63 @@
 import { gql } from '@apollo/client'
+import { eventInfo } from '../types/events.d'
+import { subscriptionInfo } from '../types/payments.d'
+import { usersInfo, memberInfo } from '../types/users.d'
 
 export const GET_USER_SESSION = gql`
-  query login($password: String = "", $username: String = "") {
-    users(where: { password: { _eq: $password }, username: { _eq: $username } }) {
-      username
-      type
-      is_active
-      id
-      created_at
-      avatar_url
+  query login($password: String!, $memberCode: String!) {
+    users(
+      where: {
+        _and: {
+          type: { _in: ["ADMIN", "SUPER_ADMIN", "TEST_ADMIN"] }
+          member_code: { _eq: $memberCode }
+          password: { _eq: $password }
+          is_active: { _eq: true }
+        }
+      }
+    ) {
+      ${usersInfo}
+      member_information {
+        ${memberInfo}
+      }
     }
   }
```

Comment:
You're already filtering the users to get just the active users, I think is not needed to check here: 

https://github.com/Azordev/did-admin-panel/blob/b0660e259cbc9c9b4afa8bc028255306de16619e/views/Login/index.tsx#L32-L36

Reply:
You right, removed condition on Login